nibabel/tests/test_scripts.py

```diff
@@ -68,7 +68,15 @@ def check_nib_ls_example4d(opts=[], hdrs_str=""):
 def test_nib_ls():
     yield check_nib_ls_example4d
     yield check_nib_ls_example4d, \
-          ['-H', 'dim,bitpix'], " \[  4 128  96  24   2   1   1   1\] 16"
+        ['-H', 'dim,bitpix'], " \[  4 128  96  24   2   1   1   1\] 16"
+    yield check_nib_ls_example4d, ['-c'], "", " 2:3 3:2 4:1 5:1.*"
```

Comment:
Yes, the yields mean that you still run further tests if one fails, but given we clean up our test errors quickly, that isn't worth that much.  So, I just do multiple `assert_true(something)` etc in the same test function.